Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -51,15 +53,63 @@
 DEFAULT_DAYSTOKEEPLOGS = 15
 
 
+@deprecated("Please use condorIDAndPathToResultFromJobRef")
 def condorIDFromJobRef(jobRef):
-  """return tuple of "jobURL" and condorID from the jobRef string"""
-  jobURL = jobRef.split(":::")[0]
-  condorID = jobURL.split("/")[-1]
+  """
+  Extract tuple of jobURL and jobID from the jobRef string.
+
+  :params jobRef: PilotJobReference of the following form: htcondorce://<ceName>/<pathToResult>-<condorID>
```

Comment:
This has to be `:param ` instead of `:params ` (and below)
Then simple types can be put as the second argument for `:param` instead of an extra line of `:type`
And it should be `str`, which is the type, instead of `string` which is a module.